inc/saiacl.h

```diff
@@ -1985,27 +2027,80 @@ typedef sai_status_t (*sai_get_acl_range_attribute_fn)(
+        _In_ const sai_attribute_t *attr);
+
+/**
+ * @brief Get ACL table group attribute
+ *
+ * @param[in] acl_table_group_id ACL table group id
+ * @param[in] attr_count Number of attributes
+ * @param[out] attr_list Array of attributes
+ *
+ * @return #SAI_STATUS_SUCCESS on success Failure status code on error
+ */
+typedef sai_status_t (*sai_get_acl_table_group_attribute_fn)(
+        _In_ sai_object_id_t acl_table_group_id,
+        _In_ uint32_t attr_count,
+        _Out_ sai_attribute_t *attr_list);
+
+/**
  * @brief Port methods table retrieved with sai_api_query()
  */
 typedef struct _sai_acl_api_t
 {
-    sai_create_acl_table_fn             create_acl_table;
-    sai_remove_acl_table_fn             remove_acl_table;
-    sai_set_acl_table_attribute_fn      set_acl_table_attribute;
-    sai_get_acl_table_attribute_fn      get_acl_table_attribute;
-    sai_create_acl_entry_fn             create_acl_entry;
-    sai_remove_acl_entry_fn             remove_acl_entry;
-    sai_set_acl_entry_attribute_fn      set_acl_entry_attribute;
-    sai_get_acl_entry_attribute_fn      get_acl_entry_attribute;
-    sai_create_acl_counter_fn           create_acl_counter;
-    sai_remove_acl_counter_fn           remove_acl_counter;
-    sai_set_acl_counter_attribute_fn    set_acl_counter_attribute;
-    sai_get_acl_counter_attribute_fn    get_acl_counter_attribute;
-    sai_create_acl_range_fn             create_acl_range;
-    sai_remove_acl_range_fn             remove_acl_range;
-    sai_set_acl_range_attribute_fn      set_acl_range_attribute;
-    sai_get_acl_range_attribute_fn      get_acl_range_attribute;
-
+    sai_create_acl_table_fn                 create_acl_table;
+    sai_remove_acl_table_fn                 remove_acl_table;
+    sai_set_acl_table_attribute_fn          set_acl_table_attribute;
+    sai_get_acl_table_attribute_fn          get_acl_table_attribute;
+    sai_create_acl_entry_fn                 create_acl_entry;
+    sai_remove_acl_entry_fn                 remove_acl_entry;
+    sai_set_acl_entry_attribute_fn          set_acl_entry_attribute;
+    sai_get_acl_entry_attribute_fn          get_acl_entry_attribute;
+    sai_create_acl_counter_fn               create_acl_counter;
+    sai_remove_acl_counter_fn               remove_acl_counter;
+    sai_set_acl_counter_attribute_fn        set_acl_counter_attribute;
+    sai_get_acl_counter_attribute_fn        get_acl_counter_attribute;
+    sai_create_acl_range_fn                 create_acl_range;
+    sai_remove_acl_range_fn                 remove_acl_range;
+    sai_set_acl_range_attribute_fn          set_acl_range_attribute;
+    sai_get_acl_range_attribute_fn          get_acl_range_attribute;
+    sai_create_acl_table_group_fn           create_acl_table_group;
+    sai_remove_acl_table_group_fn           remove_acl_table_group;
+    sai_set_acl_table_group_attribute_fn    set_acl_table_group_attribute;
+    sai_get_acl_table_group_attribute_fn    get_acl_table_group_attribute;
```

Comment:
since the relation between acl_table_group and acl_table is similar to nexthopgroup and nexthop, we also need similiar API as nexthgroup.

In this case, we also need bring acl_table_group_member concept. 

We need create/remove/set/get APi for acl_table_group_member so that we can add/remove the ACL table to ACL table group.